lib/sogou/search/api/service/keyword.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../core/base_service'
+
+module Sogou
+  module Search
+    module Api
+      module Service
+        class Keyword < Core::BaseService
+          def initialize
+            super('http://api.agent.sogou.com:8080/sem/sms/v1/CpcService?wsdl')
```

Comment:
Do you think we should put all these URL somewhere? Maybe refactor later?

Reply:
oh yes, I will refactor this bit later, probably api endpoint builder, to return relevant endpoint based on service / envrionment